File: Zadania/2025/Zadanie 3 - start/Java/Auto/Main.java
package Auto;

public class Main {

    public static void main(String[] args) {
        
        Kolo kolo = new Kolo(89, "wysoki");
        Silnik silnik = new Silnik(1499, "spalinowy");
        SkrzynaBiegow skrzynaBiegow = new SkrzynaBiegow(6, true);
        Kierownica kierownica = new Kierownica(true);

        Auto auto = new Auto(kolo, silnik, skrzynaBiegow, kierownica);

    }
    
}

File: Zadania/2025/Zadanie 3 - start/Python/Auto/Main.py
from auto import Auto
from kolo import Kolo
from silnik import Silnik
from kierownica import Kierownica
from skrzynia_biegow import SkrzyniaBiegow


def Main():
    new_Auto = Auto (
        kolo = Kolo(78 , "sredni") , 
        silnik = Silnik(1.9 , "elektryk") , 
        skrzyniaBiegow= SkrzyniaBiegow(5 , True) , 
        kierownica= Kierownica(False)
        )
    print(new_Auto)

Main()
